superset-frontend/src/SqlLab/components/QueryTable/index.tsx

```diff
@@ -266,35 +290,32 @@ const QueryTable = props => {
               )}
               placement="top"
             >
-              <Icons.Edit iconSize="small" />
+              <Icons.Edit iconSize="s" />
             </StyledTooltip>
             <StyledTooltip
               onClick={() => openQueryInNewTab(query)}
               tooltip={t('Run query in a new tab')}
               placement="top"
             >
-              <Icons.PlusCircleOutlined
-                iconSize="x-small"
-                css={verticalAlign}
-              />
+              <Icons.PlusCircleOutlined iconSize="s" css={verticalAlign} />
```

Comment:
`xs` looks to be a valid size in the typography.sizes. I think it would be safe to add `xs` to the list here: superset-frontend/src/components/Icons/IconType.ts

Reply:
xs is added to  superset-frontend/src/components/Icons/IconType.ts;  'xs' updated on line 305; Thank you